subsys/bluetooth/services/Kconfig

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019 Nordic Semiconductor ASA
+# Copyright (c) 2019 - 2020 Nordic Semiconductor ASA
```

Comment:
I think you don't need to modify this year here

Reply:
Done.